packages/api-providers/src/abstracts/relayer/types.ts

```diff
@@ -70,6 +72,44 @@ export type RelayerConfig = {
   endpoint: string;
 };
 
+export type MixerQuery = {
+  amount: number;
+  tokenSymbol: string;
+};
+
+export type RelayedChainInput = {
+  endpoint: string;
+  name: string;
+  baseOn: RelayerCMDBase;
+  // TODO: change to just contract
```

Comment:
Do we want to handle this now?

Reply:
I don't think 'contract' is any more descriptive. I removed the TODO comment.  I suppose it was there because substrate won't have a contractAddress for relayedWithdraw, but instead a treeId. Maybe it can change to `anchorIdentifier`? Any opinions @AhmedKorim ?